Translation update for Finnish

Lam4hound opened the pull request:
Lots of new strings translated to Finnish

<!--- Provide a general summary of your changes in the Title above -->

<!--- Pull requests for translations can skip the following, simply title
      your pull request as "Translation update for [language]" and squash
      your commits into a single one with the same title, if possible. -->

## Description
<!--- Describe your changes in detail -->

## Related Issue
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Translation

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Lam4hound committed:
Updated Finnish translation

Lots of new strings translated to Finnish

scakemyer commented:
Build failling, click on Details or the little red X.
```
Checking resources/language/Finnish/strings.po                         [ FAIL ]
  resources/language/Finnish/strings.po:488: keyword "Tv" unknown
  resources/language/Finnish/strings.po:488:10: syntax error
  resources/language/Finnish/strings.po:488: keyword "sarjat" unknown
  resources/language/Finnish/strings.po:488: keyword "osion" unknown
  resources/language/Finnish/strings.po:488: keyword "valikoiden" unknown
  resources/language/Finnish/strings.po:488: keyword "oletusn" unknown
  resources/language/Finnish/strings.po:488: keyword "kym" unknown
  xgettext: found 7 fatal errors
```

scakemyer commented on resources/language/Finnish/strings.po:
```diff
@@ -434,6 +455,62 @@ msgctxt "#30113"
+
+msgctxt "#30143"
+msgid "Default view for menus in TV Shows"
+msgstr Tv-sarjat-osion valikoiden oletusnäkymä""
```
Failing here.

Lam4hound committed:
Fixed the problem with the file

The translation #30143 wasn't inside quotation marks and that caused an error. Also added a couple of new translations.
